### PR TITLE
implemented fetcher

### PR DESCRIPTION
**Pull request by Tridentinus**

### CODE REVIEW

File: .gitignore
Comment (rtjord) [inline .gitignore:1]: node_modules/ is already at the top of the .gitignore. Does this do something different?

File: src/graphqlClient.js
Comment (rtjord) [inline src/graphqlClient.js:1]: Did you mean to include this file?

File: src/main.js
Comment (rtjord) [inline src/main.js:1]: Did you mean to include this file? Also, which TypeScript file was used to generate this?

File: src/myLog.log
Comment (rtjord) [inline src/myLog.log:1]: I think we should add log files to the .gitignore

**Review by rtjord — CHANGES_REQUESTED** [review]
I don't think we should include JS files and log files in our commits. When you build the project, you can tell it to put the build files in the dist/ directory, which is not tracked by git. Also, we should add log files to the gitignore.

File: src/myLog.log
Comment (Tridentinus) [reply src/myLog.log:1]: ok sure

File: .gitignore
Comment (Tridentinus) [reply .gitignore:1]: no clue I added it to bottom because I wanted to be safe

**Review by Tridentinus — COMMENTED** [review]
sounds good

**Comment by Tridentinus** [comment]
ok I made

**Review by rtjord — APPROVED** [review]

**Pull request merged**